src/Rest/RestApiBrowser.php

```diff
@@ -48,19 +79,26 @@ public function getRequestHeaders()
     }
 
     /**
+     * @return HttpClient
+     */
+    public function getHttpClient()
+    {
+        return $this->httpClient;
+    }
+
+    /**
      * @param string $method
      * @param string $url
      * @param PyStringNode $body
-     * @param array $options
      */
-    public function sendRequest($method, $url, $body = null, array $options = array())
+    public function sendRequest($method, $url, $body = null)
```

Comment:
Removing `$options` means we step back on functionality. We need to think about an alternative to configure the http adapter we choose.